recipes/collector-sidecar/recipe32.rb

```diff
@@ -0,0 +1,33 @@
+require_relative '../tools'
+
+class GraylogSidecar < FPM::Cookery::Recipe
+  description 'Graylog collector sidecar'
+
+  name     'graylog-sidecar'
+  version  data.version
+  revision data.revision
+  homepage 'https://graylog.org'
+  arch     'i386'
+
+  source data.source
+  sha256 data.sha256
+
+  maintainer 'Graylog, Inc. <hello@graylog.org>'
+  vendor     'graylog'
+  license    'GPLv3'
+
+  config_files '/etc/graylog/sidecar/sidecar.yml'
+
+  fpm_attributes rpm_os: 'linux'
+
+  def build
+  end
+
+  def install
+    bin.install "#{version}/linux/386/graylog-sidecar"
+    etc('graylog/sidecar').install '../../sidecar-example.yml', 'sidecar.yml'
```

Comment:
You can use the `workdir` helper here to avoid the relative path. The `workdir` helper takes the recipe directory and appends the given path arguments:

```suggestion
    etc('graylog/sidecar').install workdir('sidecar-example.yml'), 'sidecar.yml'
```

Reply:
Thanks. Should be fixed now.